### ISSUE TITLE
Polymer Dart app not compiling link imports and throws CSP issues

### ISSUE THREAD

**Issue by mit-mit** [dart-lang/polymer-dart#566]
_From @DartBot on October 7, 2014 4:23_

*This issue was originally filed by ba...&#064;vt.edu*
_____
**What steps will reproduce the problem?**
1. Create a new chrome packaged app in Dart Editor
2. Add polymer and paper elements dependency to pubspec and build
3. Create a new element in separate file and call it in the main file.

**What is the expected output? What do you see instead?**
The main file cannot recognize the file in the folder. Chrome console says &quot;File not found&quot;

**What version of the product are you using?**
Dart Editor and SDK 1.6
**On what operating system?**
Mac OSX
**What browser (if applicable)?**
Chrome App on Chrome 37

**Please provide any additional information below.**

_Copied from original issue: dart-lang/sdk#21256_

**Comment by mit-mit** [comment]
_From @sigmundch on October 7, 2014 20:7_

Can you please provide more details to help us reproduce the issue you are seeing?

There are many details that are hard to imagine without the code, like how are you importing/calling elements, what does your pubspec look like, and how do you launch the app.

If you can actually include a link to the code you are using, that would be the best. 
______
*Added [NeedsInfo](../labels/NeedsInfo) label.*

**Comment by mit-mit** [comment]
_From @DartBot on October 9, 2014 20:12_

*This comment was originally written by ba...&#064;vt.edu*
_____
I have attached a sample app that I created. It reproduces the issue. Note that I compiled it to JS and ran it from build folder. Dart app recognizes the file but still has CSP issues, but the JS app has both issues.
______
**Attachment:**
[interest_me_dart.zip](https://storage.googleapis.com/google-code-attachments/dart/issue-21256/comment-2/interest_me_dart.zip) (6.80 MB)

**Comment by mit-mit** [comment]
_From @sigmundch on October 9, 2014 20:34_

From a quick look I recognize two bits that might be the reason behind the problem you are seeing:

1. Your pubspec says:

- polymer:
&nbsp;&nbsp;&nbsp;&nbsp;entry_points: web/index.html

but it should be the name of your file under web:

- polymer:
&nbsp;&nbsp;&nbsp;&nbsp;entry_points: web/interest_me_dart.html

2. I've never used the "chrome" transformer before, but it seems to do something that the "polymer" transformer also does internally. I'm not sure how well they play with each other, so it might be better to remove it entirely and let polymer take care of it.

I'm not sure how to run the actual generated code, but after I did the two changes above, the output seems closer to what I'd expect. Let us know if that fixes your problem!

**Comment by mit-mit** [comment]
_From @DartBot on October 9, 2014 21:22_

*This comment was originally written by ba...&#064;vt.edu*
_____
Thanks for the help. I din't see that. But, there is still some problem. First, I ran the dart app with dartium, but I get a load of CSP issues, and no output. Second, I tried compiling to JS and it seems to work without errors, but I don't see the custom element tag rendered. The tag is empty when I inspect it.
